app/javascript/techradar.js

```diff
@@ -55,76 +63,131 @@
                     });
 
                 // creates all spots
-                createRadarSpots(enterElement);
+                createRadarSpots(enterElement, quadrant);
 
                 // write a head-line per quadrant
                 createListHeadline(quadrant, (quadrantIndex < 2) ? 0 : 1);
                 createSpotLabelAndListElements(enterElement, quadrant);
             }, this);
 
             renderHelp(canvas, this.radar);
-            
-            function sortByArcOrder(a,b) {
+            renderTooltip(canvas);
```

Comment:
Why isn't the tooltip rendered on a per-spot basis - I mean I'd expected to see this kind of call in the `createRadarSpots`-function? Obviously it works, but in terms of maintainability IMHO we should think about it again.